#!/usr/bin/env python
PACKAGE = "focus"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kp_rad", double_t, 0, "kernel excitatory radius", .05, 0.01, 1)
gen.add("ki_ratio", double_t, 0, "kernel inhibition distance rate", 5, 0, 50)
gen.add("min_input", double_t, 0, "input intensity saturation min", 0, 0, 255)
gen.add("max_input", double_t, 0, "input intensity saturation max", 20, 0, 255)
gen.add("shift_rad", double_t, 0, "max small shift radius", 0.2, 0.0, 1.0)
gen.add("shift_freeze", double_t, 0, "shift freeze duration", 0.1, 0.0, 2.0)


exit(gen.generate(PACKAGE, "focus", "LocusParam"))
